backend/apps/volontulo/forms.py

```diff
@@ -119,6 +119,52 @@ class Meta(object):
             'volunteers_limit',
         ]
 
+    def clean(self):
+        super(CreateOfferForm, self).clean()
+        started_at = self.cleaned_data.get('started_at')
```

Comment:
This method is very long. You could refactor it's content into 3 methods : `_clean_start_finish(self)`, `_clean_recruitment_start_finish(self)` and `_clean_reserve_recruitment_start_finish(self)`. Then in `clean(self)` you could call them:
```python
def clean(self):
    super(CreateOfferForm, self).clean()
    self._clean_start_finish(self)
    self._clean_recruitment_start_finish(self)
    self._clean_reserve_recruitment_start_finish(self)
```
After that you can write a test for each of these methods.